.github/workflows/check-contrib.yml

```diff
@@ -0,0 +1,83 @@
+name: check-contrib
+
+on: [push, pull_request]  # would you need to define PRs containing .eb files?
+
+permissions:
+  contents: read  # to fetch code (actions/checkout)
+
+jobs:
+  check-contrib:
+    name: Run EasyBuild check-contrib
+    runs-on: ubuntu-22.04
+
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2  # shallow but deep enough to diff last commit; I don't want to have do to fetch-depth: 0 and get the full history
+
+      - name: Set up Python env
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.8'
+
+      - name: Upgrade pip
+        run: python -m pip install --upgrade pip
+
+      - name: Install EasyBuild and required deps
+        run: |
+          echo "Installing easybuild and pycodestyle"
+          pip install easybuild==4.9.4 pycodestyle
+
+          echo "Installing LMOD"
+          sudo apt-get install -y lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev
```

Comment:
This install section is a bit messy. It can be tidied up. Add neater lines to a variable and then 

$APT_APPS_INSTALL _LIST= lua5.2 liblua5.2-dev lua-filesystem lua-posix tcl tcl-dev curl
sudo apt-get install -y $APT_APPS_INSTALL_LIST